lms/djangoapps/teams/static/teams/js/views/teams.js

```diff
@@ -4,34 +4,45 @@
         'backbone',
         'teams/js/views/team_card',
         'common/js/components/views/paginated_view',
-        'teams/js/views/team_actions'
-    ], function (Backbone, TeamCardView, PaginatedView, TeamActionsView) {
+        'text!teams/templates/team-actions.underscore'
+    ], function (Backbone, TeamCardView, PaginatedView, teamActionsTemplate) {
         var TeamsView = PaginatedView.extend({
             type: 'teams',
 
+            events: {
+                'click a.browse-teams': 'browseTeams',
+                'click a.search-team-descriptions': 'searchTeamDescriptions',
```

Comment:
FYI, that search (within a selected topic) will search across Team names, descriptions, country, and language.